### ISSUE TITLE
[REQUEST]: Configurable size limits

### ISSUE THREAD

**Issue by ThatXliner** [ThatXliner/aioudp#1]
### Prerequisites

- [X] I have searched the [issues](https://github.com/ThatXliner/aioudp/issues) and believe that it has not already been requested (and possibly rejected)
- [X] I have made sure that this feature isn't possible with this project's plugin system and/or configuration and core changes must be made to make this happen
- [X] I agree to follow the [Code of Conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct/)

### Feature description

Basically, be able to configure size limits to the internal queues and buffers. Maybe add download/upload throttling like [aio-udp-server](https://github.com/bashkirtsevich-llc/aioudp)
